index.csv

```diff
@@ -224,12 +228,12 @@ whatwg-fetch,MIT,0.5.0,0.6.0,0.6.1,0.7.0,0.8.0,0.8.1,0.8.2,0.9.0,0.10.0,0.10.1,0
 when,MIT,1.2.0,1.3.0,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.5.0,1.5.1,1.5.2,1.6.0,1.6.1,1.7.0,1.7.1,1.8.0,1.8.1,2.0.0,2.0.1,2.1.0,2.1.1,2.2.0,2.2.1,2.3.0,2.4.0,2.5.0,2.5.1,2.6.0,2.7.0,2.7.1,2.8.0,3.0.0,3.0.1,3.1.0,3.2.0,3.2.1,3.2.2,3.2.3,3.3.0,3.3.1,3.4.0,3.4.1,3.4.2,3.4.3,3.4.4,3.4.5,3.4.6,3.5.0,3.5.1,3.5.2,3.6.0,3.6.1,3.6.2,3.6.3,3.6.4,3.7.0,3.7.1,3.7.2,3.7.3
 winston,MIT,0.7.3,0.8.0,0.8.1,0.8.2,0.8.3,0.9.0,1.0.0
 wordwrap,MIT,0.0.2
-wrench,MIT,0.1.0,1.0.0,1.1.0,1.2.0,1.3.0,1.3.1,1.3.2,1.3.3,1.3.4,1.3.5,1.3.6,1.3.7,1.3.8,1.3.9,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.5.0,1.5.1,1.5.3,1.5.4,1.5.5,1.5.6,1.5.7,1.5.8,1.5.9
+wrench,MIT,0.1.0,0.4.0,0.4.1,0.4.2,0.4.3,0.4.4,0.4.5,0.4.6,0.4.7,0.4.8,1.0.0,1.1.0,1.2.0,1.3.0,1.3.1,1.3.2,1.3.3,1.3.4,1.3.5,1.3.6,1.3.7,1.3.8,1.3.9,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.5.0,1.5.1,1.5.3,1.5.4,1.5.5,1.5.6,1.5.7,1.5.8,1.5.9
```

Comment:
cc @ryanmcgrath

Reply:
Wrench is deprecated and has been for years, and I'm not interested in updating it. What exactly is this for...?

Reply:
It doesn't need any updates :-) all i'd need is your explicit confirmation that indeed, the MIT license should apply to all the listed versions of wrench.

Reply:
If it's there and lasted the duration of the project, then yes, it applies. :)